data/web/inc/init_db.inc.php

```diff
@@ -864,7 +864,7 @@ function init_db_schema() {
       "sogo_sessions_folder" => array(
         "cols" => array(
           "c_id" => "VARCHAR(255) NOT NULL",
-          "c_value" => "VARCHAR(255) NOT NULL",
+          "c_value" => "VARCHAR(4096) NOT NULL", 

```

Comment:
there is some white space at the end of this line added

Reply:
Oh yes i saw that. Will fix it